spec/game_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'spec_helper'
+
+describe Game do
+  before(:each) do
+    @game = Game.new(false, Date.new(2021, 5, 5))
+  end
+
+  it 'is an instance of Game' do
+    expect(@game).to be_an_instance_of(Game)
+  end
+
+  it 'has a last played date' do
+    expect(@game.last_played_at).to eql(Date.new(2021, 5, 5))
+  end
```

Comment:
- [ ] kindly add a test for all public methods like `can_be_archived?` and the rest